lib/tasks/db.rake

```diff
@@ -1,37 +0,0 @@
-namespace :db do
-  task remove_unused_fields_from_assets: :environment do
-    if Asset.where(access_limited: true).none?
-      Asset.update_all("$unset" => { "access_limited" => true })
-      puts "Asset#access_limited field removed successfully"
-    else
-      puts "Error: Unable to remove Asset#access_limited"
-    end
-
-    if Asset.where(:organisation_slug.ne => nil).none?
-      Asset.update_all("$unset" => { "organisation_slug" => true })
-      puts "Asset#organisation_slug field removed successfully"
-    else
-      puts "Error: Unable to remove Asset#organisation_slug"
-    end
-  end
-
-  desc "Set the size field for all assets from the etag"
-  task set_size_for_all_uploaded_assets: :environment do
```

Comment:
Looks like https://github.com/alphagov/asset-manager/commit/16e323e315264bc8fc662174fd0bc30f1e202e1e added a few methods which could be removed too. If they can't, it would be good to write a brief note explaining why.

Reply:
Good spot. I've updated the commit.